instance/models/openedx_instance.py

```diff
@@ -160,10 +167,41 @@ def spawn_appserver(self):
             # Copy the current value of each setting into the AppServer, preserving it permanently:
             configuration_database_settings=self.get_database_settings(),
             configuration_storage_settings=self.get_storage_settings(),
+            configuration_extra_settings=self.get_extra_settings(),
             **instance_config
         )
         if app_server.provision():
             # If the AppServer provisioned successfully, make it the active one:
             # Note: if I call spawn_appserver() twice, and the second one provisions sooner, the first one may then
             # finish and replace the second as the active server. We are not really worried about that for now.
             self.set_appserver_active(app_server.pk)
+
+
+class OpenEdXProductionInstance(OpenEdXInstance):
+    """
+    OpenEdXProductionInstance: An OpenEdXInstance using production settings
+    """
+
+    LATEST_RELEASE = "named-release/dogwood.rc"
+
+    def save(self, **kwargs):
```

Comment:
@antoviaque I should have mentioned on the ticket that I don't intend to keep this class :) It is part of the approach that I implemented yesterday (in https://github.com/open-craft/opencraft/pull/71/commits/cc963ecddd47a313d4c31477099e5acc3c2f2e09). The reason I didn't remove it yet was to make it easier to review the second approach that I implemented today (in https://github.com/open-craft/opencraft/pull/71/commits/5eb1d3227bcb0d544f8df6f4415335d9e5601018).